js/layout-javascript/small-card-code.js

```diff
@@ -333,6 +333,8 @@ DynamicList.prototype.attachObservers = function() {
         return;
       }
 
+      $(event.target).parents('.small-card-list-wrapper').addClass('hidden');
```

Comment:
`$(event.target)` and `$(this)` are used multiple times in the same closure. Use a variable.

Reply:
@tonytlwu Fixed